### ISSUE TITLE
Test questionaries on a live globaleaks system

### ISSUE THREAD

**Issue by fpietrosanti** [catskillmarina/violencetracker#1]
This ticket is to test questionaries on a live globaleaks system, to see how those behave and which could be the right area of improvements.

Following test-implementation of questionnaires, we shall iterate to understand which kind of globaleaks improvements are needed to make it suitable for violence tracker project

**Comment by catskillmarina** [comment]
The issue that first came to mind is the possibility that there might be many assailants per each incident. Perhaps one to many relationships can be dealt with  using ajax. If another assailant section is needed - click a + to create another section.  Globaleaks does not need to deal with the full complexities of data normalization, only getting the data to the reporting agency safely and securely.
